src/name/admitriev/jhelper/network/SimpleHttpServer.java

```diff
@@ -33,8 +34,14 @@ public void run() {
 				if (serverSocket.isClosed()) {
 					return;
 				}
-				try (Socket socket = serverSocket.accept()) {
+				try (Socket socket = serverSocket.accept();
+					 PrintWriter serverOut = new PrintWriter(
+							 new OutputStreamWriter(socket.getOutputStream(), "UTF-8"), true)) {
+
 					InputStream inputStream = socket.getInputStream();
+					// Make a response
+					serverOut.println(RESPONSE_HEADERS + RESPONSE_CONTENT.length() + RESPONSE_END_OF_HEADERS + RESPONSE_CONTENT);
```

Comment:
Is there something like python's format? 
Let's make it more obvious that content length goes to Content-Length header

Reply:
Do you mean something like this ?

```suggestion
					serverOut.println(String.format(RESPONSE_HEADERS, RESPONSE_CONTENT.length()) + RESPONSE_END_OF_HEADERS + RESPONSE_CONTENT);
```